contracts/utils/FuseFixer.sol

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
To be consistent, would use `import {} from "@openzeppelin/" syntax for these imports too